app/grants/models.py

```diff
@@ -567,6 +567,18 @@ def get_hash_from_web3(self):
             args['nonce'],
             ).call()
 
+    @property
+    def value_usdt(self):
+        try:
+            return float(convert_amount(
```

Comment:
@captnseagraves wrote this code. i just refactored it so that it was DRY.

i agree with your comment that it should be time-pegged, similar to how `bounty.value_in_usdt_then` works